dockers/docker-sonic-telemetry/telemetry.sh

```diff
@@ -63,10 +63,10 @@ if [ -z $CLIENT_AUTH ] || [ $CLIENT_AUTH == "false" ]; then
 fi
 
 LOG_LEVEL=$(echo $GNMI | jq -r '.log_level')
-if [ ! -z $LOG_LEVEL ]; then
-    TELEMETRY_ARGS+=" -v=$LOG_LEVEL"
-else
+if [ -z $LOG_LEVEL ] || [ $LOG_LEVEL == "null" ]; then
```

Comment:
>$LOG_LEVEL [](http://example.com/codeflow?start=26&length=10)

If you expect only numbers, check for numbers as allowlist. ref: [https://stackoverflow.com/a/2210386/2514803](https://stackoverflow.com/a/2210386/2514803)

Reply:
Done